app/src/js/components/Granules/list.js

```diff
@@ -115,7 +116,7 @@ class AllGranules extends React.Component {
     const query = this.generateQuery();
     const view = this.getView();
     const displayCaseView = displayCase(view);
-    const statusOptions = (view === 'all') ? statusOptions : null;
+    const statOptions = (view === 'all') ? statusOptions : null;
```

Comment:
Can you change statOptions to statusOpts?  stat is easily confused with statistics.

Reply:
Done!